pkg/nsx/services/securitypolicy/firewall.go

```diff
@@ -12,26 +11,26 @@ import (
 	"github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha1"
 	"github.com/vmware-tanzu/nsx-operator/pkg/logger"
 	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/services/common"
-	"github.com/vmware-tanzu/nsx-operator/pkg/util"
 )
 
 var (
-	log                       = logger.Log
-	MarkedForDelete           = true
-	EnforceRevisionCheckParam = false
-	Converter                 *bindings.TypeConverter
+	log                        = logger.Log
+	MarkedForDelete            = true
+	EnforceRevisionCheckParam  = false
+	ResourceTypeSecurityPolicy = common.ResourceTypeSecurityPolicy
+	ResourceTypeRule           = common.ResourceTypeRule
+	ResourceTypeGroup          = common.ResourceTypeGroup
+	NewConverter               = common.NewConverter
+	storeSecurityPolicy        common.Store
+	storeGroup                 common.Store
+	storeRule                  common.Store
```

Comment:
securityPolicyComStore        common.Store
groupComStore                    common.Store
ruleComStore                      common.Store
maybe be better to say these three var whose type common.store, but dynamically pointing to common.Store specific interface implementation in code line #L51 below

Reply:
ok